src/variable.c

```diff
@@ -1128,6 +1129,14 @@ mrb_class_find_path(mrb_state *mrb, struct RClass *c)
   return path;
 }
 
+mrb_int
+mrb_obj_iv_tbl_memsize(mrb_state* mrb, mrb_value obj)
+{
+  return sizeof(iv_tbl) +
+    (sizeof(segment) * ceil(iv_size(mrb, mrb_obj_ptr(obj)->iv)/
```

Comment:
Because of `MRB_WITHOUT_FLOAT`, it needs to replace the `<math.h>` header file and the `ceil()` function.

Reply:
Thank you. I forget that we sometimes do not have floats.